CHANGELOG.md

```diff
@@ -14,6 +14,9 @@ and what APIs have changed, if applicable.
 
 ## [Unreleased]
 
+## [29.18.12] - 2021-05-26
```

Comment:
With new changes, do we need a new minor version?

Reply:
I don't think we need a minor version bump. All changes in this PR are backward compatible